### DISCUSSION TITLE
Bit Nets ?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [karpathy/llm.c#110] [Ideas]
@karpathy 
Maybe you want to skip over the "old school" Lama, Gemma, Mistral altogether, for which there are a lot of implementations (including similar to what you do -  llama.cpp, Mozilla's llamafile), and instead - take llm.c into a radically new and hot direction?

https://github.com/Mozilla-Ocho/llamafile/discussions/313

https://twitter.com/winglian/status/1778675583817326842?t=v7dewH1RQp-mhkeR97KRog&s=19

**Comment by karpathy** [comment]
haha probably no but love the thought!
